src/lsst/rsp/startup/services/labrunner.py

```diff
@@ -772,19 +706,14 @@ def _set_timeout_variables(self) -> list[str]:
     def _start(self) -> None:
         log_level = "DEBUG" if self._debug else "INFO"
         cmd = [
-            "python3",
-            "-s",
-            "-m",
-            "jupyter",
-            "labhub",
+            "jupyterhub-singleuser",
             "--ip=0.0.0.0",
             "--port=8888",
             "--no-browser",
             f"--notebook-dir={self._home!s}",
-            f"--hub-prefix={self._stash['jupyterhub_path']}",
-            f"--hub-host={self._stash['external_host']}",
             f"--log-level={log_level}",
             "--ContentsManager.allow_hidden=True",
+            f"--ContentsManager.root_dir={self._home!s}",
```

Comment:
Why is this now needed?

Reply:
Not only is it not, that's apparently a setting that is no longer respected.  It was part of the jupyterhub-singleuser change and maybe it was needed back in the JL 2.x/early 3.x days?